packages/pcre.rb

```diff
@@ -3,41 +3,43 @@
 class Pcre < Package
   description 'The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax and semantics as Perl 5.'
   homepage 'http://pcre.org/'
-  version '8.44-1'
+  @_ver = '8.44'
+  version "#{@_ver}-1"
```

Comment:
Shouldn't this be `-2`?  I guess it doesn't matter now that you overwrote the previous binaries on Bintray.  But...now it's difficult to `crew upgrade`.

Reply:
Never mind.  This is a new package that was never merged.

Reply:
Never mind.  I guess the update to `-1` wasn't merged yet.